js/web/lib/wasm/jsep/init.ts

```diff
@@ -61,7 +61,8 @@ class ComputeContextImpl implements ComputeContext {
     return this.backend.currentKernelCustomData;
   }
   get customDataBuffer(): Uint8Array {
-    return this.module.HEAPU8.subarray(this.customDataOffset, this.customDataOffset + this.customDataSize);
+    return this.module.HEAPU8.subarray(
+        this.customDataOffset >>> 0, (this.customDataOffset >>> 0) + this.customDataSize);
```

Comment:
This should be no necessary as value of `this.customDataOffset` comes from HEAPU32

Reply:
done